webapp/src/apis/intersections/events-api.ts

```diff
@@ -9,6 +9,7 @@ const EVENT_TYPES: Item[] = [
   { label: 'ConnectionOfTravelEvent', value: 'connection_of_travel' },
   { label: 'IntersectionReferenceAlignmentEvent', value: 'intersection_reference_alignment' },
   { label: 'LaneDirectionOfTravelEvent', value: 'lane_direction_of_travel' },
+  //   { label: "ProcessingTimePeriod", value: "processing_time_period" },
```

Comment:
Is the commented out code intended to be kept?

Reply:
Changed to a TODO comment